src/TesterInternal/GeoClusterTests/TestingClusterHost.cs

```diff
@@ -0,0 +1,251 @@
+﻿using Orleans.Runtime;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Orleans.TestingHost;
+using System.Reflection;
+using System.Globalization;
+using UnitTests.Tester;
+using Orleans.Runtime.Configuration;
+using System.Net;
+using System.Net.Sockets;
+using Orleans;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Tests.GeoClusterTests
+{
+    /// <summary>
+    /// A utility class for tests that include multiple clusters.
+    /// It calls static methods on TestingSiloHost for starting and stopping silos.
+    /// </summary>
+    public class TestingClusterHost   
```

Comment:
Short answer: we wanted to change as little as possible in TestingSiloHost so as to minimize interference with existing tests. Longer answer: TestingSiloHost will create only a single cluster when constructed, and we cannot use multiple instances of TestingSiloHost because it uses static fields (not sure why). But TestingSiloHost did already encapsulate all relevant functionality for starting and stopping of silos in static methods so we could easily use those to implement TestingClusterHost.